server/__tests__/configuration.test.js

```diff
@@ -397,4 +476,6 @@ describe('Test Meilisearch plugin configurations', () => {
 
     expect(contentTypes).toEqual(['restaurant', 'about'])
   })
+
+  // TODO: add tests here
```

Comment:
👀

Reply:
I added the tests 😂 Removing this comment